modules/tinymce/CHANGELOG.md

```diff
@@ -57,6 +57,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Moved the `print` plugin's functionality to TinyMCE core #TINY-8314
 - Renamed the `getShortEndedElements` Schema API to `getVoidElements` #TINY-8344
 - Changed the default statusbar element path delimiter from `»` to `›` #TINY-8372
+- Renamed `font_formats` option to `font_family_formats`, `fontselect` toolbar button to `fontfamily`, and `fontformats` menu item to `fontfamily` #TINY-8328
```

Comment:
I wonder if it would read better if this was separate changelog lines or maybe a markdown list if we support that?

Reply:
There's some issue with using nested lists from what I recall, as we went to use it once before and we reverted it 🤷 . I'd agree we should probably just put these on different lines as they are different things.

Reply:
Fair point, will separate them

Reply:
I'd like to dig more into why nested lists are a problem; they are very useful in large releases like this to visually group similar changes.

Reply:
I separated the lines based on what things are changed to, rather than every single items. So, `- Renamed `fontselect` toolbar button and `fontformats` menu item to `fontfamily` #TINY-8328` is still a thing. That work?

Reply:
Yeah, that works for now, my comment was more of a general statement